api/api/views/generate_teams.py

```diff
@@ -37,9 +36,8 @@ def generate_teams(self, request):
         input_data = GenerateTeamsDataLoader(request_data).load()
```

Comment:
The line above (`GenerateTeamsValidator(request_data).validate()`) has gotta get done - can be in a separate PR though, let's take things chunk by chunk

Reply:
Oops